src/Panel.vue

```diff
@@ -236,9 +236,10 @@
           this.src = this.src.split('#')[0];
         }
       }
-      
-      //Set prop data to computed prop value
-      this.expanded =  (!this.expandableBool && this.expanded !== 'false') || this.expandedBool;
+      // Edge case where user might want non-expandable panel that isn't expanded by default
+      const notExpandableNoExpand = !this.expandableBool && this.expanded !== 'false';
+      // Set prop data to computed prop value
+      this.expanded =  this.expandedBool || notExpandableNoExpand;
```

Comment:
A note here for archival purpose: The order of the boolean is creating the "minimized" bug (whereby if the panel is minimized at the start, one must click the button twice to expand the panel).

It should have been:
```
this.expanded = notExpandableNoExpand || this.expandedBool;
```

Alternatively, fix `toBoolean()` to deal with null values (`this.expandedBool` is being evaluated to `null`).